documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -306,6 +306,8 @@ spec:
     namespace: <namespace>
 EOF
 ----
+<1> Specify the URL of the API endpoint.
```

Comment:
Missing callouts text
![image](https://github.com/kubev2v/forklift-documentation/assets/106804821/6be993ba-c312-49d4-9b66-f8ad280f11bc)

@RichardHoch - I have the feeling this will fix the duplicate callouts later you note later. What do you think?